docs/modules/input.md

```diff
@@ -127,6 +127,22 @@ You can use an index from 1-5 (button 0 is invalid) or a lowercase name:
 * `x1`
 * `x2`
 
+#### `static setCursor(name: String)`
+Sets the system cursor to the name provided. If you attempt to set the cursor to an invalid name it will reset to `arrow`.
+Available cursor names are:
+* `arrow`
+* `ibeam`
+* `wait`
+* `crosshair`
+* `waitarrow`
+* `sizenwse`
+* `sizenesw`
+* `sizewe`
+* `sizens`
+* `sizeall`
+* `no`
+* `hand`
```

Comment:
Where did these names come from? I think I'd like to consider how we name them a bit more carefully, if I can make a choice :P

Reply:
These names are just the tail end of the enum definitions from SDL in lowercase. We can name them anything :)
https://wiki.libsdl.org/SDL_SystemCursor

Reply:
Happy with these names, although not all the cursors are available on every platform (mostly OSX is the problem here) so we should make a note of that in the documentation.

Reply:
I added some notes to the docs about what cursors are available on mac in 570a0a1. If you'd like to make any further changes feel free :smile:

Reply:
I'm debating whether we should specifically say what OSX does, or just be more general, to cover other platforms "These cursors may not be available on all platforms, or may display differently."